src/qml/qmlwaveformrenderer.h

```diff
@@ -397,7 +397,7 @@ class QmlWaveformRendererMark
   signals:
     void playMarkerColorChanged(const QColor&);
     void playMarkerBackgroundChanged(const QColor&);
-    void defaultMarkChanged(mixxx::qml::QmlWaveformMark*);
+    void defaultMarkChanged(QmlWaveformMark*);
```

Comment:
This required to be fully qualified prior to Qt 6.9 (see clazy warning). Shall we wrap that in preprocessor? 
```suggestion
#if QT_VERSION >= QT_VERSION_CHECK(6, 9, 0)
    void defaultMarkChanged(QmlWaveformMark*);
#else
    void defaultMarkChanged(mixxx::qml::QmlWaveformMark*);
#endif 
```

Reply:
done